spec/core/1.0.md

```diff
@@ -307,6 +307,27 @@ The name of any private module SHOULD start with two leading underscores.
 
 *Rationale:* We strive to avoid name clashes between modules by different vendors, and reserve short module names for official modules to minimize message length for common messages.
 
+### 3.4. Predefined property types
+
+```abnf
+integer          = "0" | ( 0*1sign nzdigit *digit )
+unsigned-integer = "0" | ( nzdigit *digit )
+
+sign = "+" / "-"
+```
+
+When a property is said to **accept integer values**, this means that the set of acceptable values for this property is the set of all atoms whose string value matches the `<integer>` grammar element defined above.
+Analogously, when a property is said to **accept unsigned integer values**, this means that the set of acceptable values for this property is the set of all atoms whose string value matches the `<unsigned-integer>` grammar element defined above.
+
+In both these cases, the numerical value of each such atom is that value which is returned when the `atoll()` function as defined in [the POSIX.1 specification](http://pubs.opengroup.org/onlinepubs/9699919799/) is applied to the string value of the atom.
```

Comment:
Maybe link directly to http://pubs.opengroup.org/onlinepubs/9699919799/functions/atoll.html